index.php

```diff
@@ -14,21 +15,27 @@
         </div>
         <br>
         <div class=submitForms>
-            <form action="/index.php" method="post"> 
-                <label for="brand">Brand:</label><br>
-                <input type="text" name="brand"><br><br>
-                <label for="model">Model:</label><br>
-                <input type="text" name="model"><br><br>
-                <label for="price">Price:</label><br>
-                <input type="text" name="price"><br><br>
+            <p><span class="error">* required field</span></p>
+            <form action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]);?>" method="post"> 
+                <label for="brand">Brand:</label>
+                <span class="error">* <?php echo $brandErr;?></span><br>
+                <input type="text" name="brand" value="<?php echo $brand;?>"><br><br>
+                <label for="model">Model:</label>
+                <span class="error">* <?php echo $modelErr;?></span><br>
+                <input type="text" name="model" value="<?php echo $model;?>"><br><br>
+                <label for="price">Price:</label>
+                <span class="error">* <?php echo $priceErr;?></span><br>
+                <input type="text" name="price" value="<?php echo $price;?>"><br><br>
```

Comment:
Having a required tab in your input types may have been easier but your method works well

Reply:
I learnt about the required tab after I had done it this way haha.